content/telegraf/v1.18/about_the_project/release-notes-changelog.md

```diff
@@ -7,6 +7,45 @@ menu:
     weight: 10
     parent: About the project
 ---
+
+## v1.18.1 [2021-04-07]
+
+- Agent: Closes running outputs when agent reloads on SIGHUP.
+
+### Input plugin updates
+
+- [Docker Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/docker)(`docker`):
+  Fixed panic when parsing container statistics.
+- [Exec Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/exec)(`exec`):
+  Plugin does not truncate messages in debug mode.
+- [IPMI Sensor Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ipmi_sensor)(`ipmi_sensor`):
+  Fixed panic by implementing a length check to plugin.
+- [MySQL Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql)(`mysql`):
+  Fixed the ability to handle ‘binary logs’ query for MySQL version 8.0+.
+- [NFS Client Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/nfsclient)(`nfsclient`):
+  Fixed integer overflow in fields received by mountstat.
+- [Ping Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ping)(`ping`):
+  Fixed an issue when `method=”native”` by always calling `SetPrivileged(true)`.
+- [SNMP Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/snmp)(`snmp`):
+  Fixed `init()` when no MIBs are installed.
+- [SQL Server Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver)(`sqlserver`):
+  Fixed `sqlserver_process_cpu` calculation.
+- [Tail Input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/tail)(`tail`):
+  Added configurable option to override `path` tag.
+
+### Output plugin updates
+
+- [Azure Monitor Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/azure_monitor)(`azure_monitor`):
+  Fixed an issue to handle error when initializing the authentication object.
+- [Yandex Cloud Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/yandex_cloud_monitoring) (`yandex_cloud_monitoring`):
```

Comment:
```suggestion
- [Yandex Cloud Monitoring Output](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/yandex_cloud_monitoring) (`yandex_cloud_monitoring`):
```

Reply:
Let's add space after the rest of the plugins like we do here
Yandex Cloud Output (`yandex_cloud_monitoring`):